prov/verbs/src/verbs_info.c

```diff
@@ -1086,7 +1099,13 @@ int fi_ibv_init_info(const struct fi_info **all_infos)
 {
 	struct ibv_context **ctx_list;
 	struct fi_info *fi = NULL, *tail = NULL;
-	int ret = 0, i, num_devices;
+	const struct verbs_ep_domain *ep_type[] = {
```

Comment:
This change and the subsequent change to fi_ibv_ini_info() are not directly related to adding the xrc/feature capability. I'm going to remove them from the commit and they can be considered on their own merit later.